neurodamus/utils/memory.py

```diff
@@ -198,25 +198,31 @@ def print_allocation_stats(rank_allocation, rank_memory):
 
 
 @run_only_rank0
-def export_allocation_stats(rank_allocation, filename):
+def export_allocation_stats(rank_allocation, filename, metype_memory_usage, memory_per_metype_file):
     """
-    Export allocation dictionary to serialized pickle file.
+    Export allocation dictionary to a serialized pickle file and metype memory usage to a JSON file.
     """
+    # Export rank_allocation to a compressed pickle file
     compressed_data = gzip.compress(pickle.dumps(rank_allocation))
     with open(filename, 'wb') as f:
         f.write(compressed_data)
 
+    # Export metype_memory_usage to a JSON file
```

Comment:
```suggestion
    compressed_data = gzip.compress(pickle.dumps(rank_allocation))
    with open(filename, 'wb') as f:
        f.write(compressed_data)

```
The comments just state what the code tells you without further informational content.

Reply:
Since the same function is exporting two different files in two differents format I thought they might be informative. But the code is pretty much self-explanatory. 
Will remove the comments.